bundles/org.openhab.binding.mqtt.homie/src/main/java/org/openhab/binding/mqtt/homie/generic/internal/MqttBindingConstants.java

```diff
@@ -33,4 +33,8 @@
 
     public static final String HOMIE_PROPERTY_VERSION = "homieversion";
     public static final String HOMIE_PROPERTY_HEARTBEAT_INTERVAL = "heartbeat_interval";
+
+    public static final int HOMIE_DEVICE_TIMEOUT = 5000;
+    public static final int HOMIE_SUBSCRIBE_TIMEOUT = 500;
+    public static final int HOMIE_ATTRIBUTE_TIMEOUT = 200;
```

Comment:
Nice to put them in constants. Can you add the unit on these constants:
```suggestion
    public static final int HOMIE_DEVICE_TIMEOUT_MS = 5000;
    public static final int HOMIE_SUBSCRIBE_TIMEOUT_MS = 500;
    public static final int HOMIE_ATTRIBUTE_TIMEOUT_MS = 200;
```

Reply:
Sure @Hilbrand, done in the last commit! :+1:

Reply:
Did you intentionally increase the timeout to 15 seconds or was it a mixup with some testing you did?

Reply:
Ups, I pushed the changes before writing my "longer" message, sorry. Yes, I have increased the timeout on purpose, because for really complex devices 5 seconds is not enough. That is also the timeout value that was set before #7760, so nothing unexpected. I really hope that this PR can "fix" the Homie bug that we've been suffering from for so long, and I'm really sorry for all my trial and errors, I thought that it would be easier to solve, but it seems that it wasn't true :disappointed: